internal/image/docker_schema1_test.go

```diff
@@ -269,13 +269,15 @@ func TestManifestSchema1EmbeddedDockerReferenceConflicts(t *testing.T) {
 }
 
 func TestManifestSchema1Inspect(t *testing.T) {
+
 	for _, m := range []genericManifest{
 		manifestSchema1FromFixture(t, "schema1.json"),
 		manifestSchema1FromComponentsLikeFixture(t),
 	} {
 		ii, err := m.Inspect(context.Background())
 		require.NoError(t, err)
 		created := time.Date(2018, 1, 25, 0, 37, 48, 268558000, time.UTC)
+		var annotations map[string]string
```

Comment:
Non-blocking:

If this is only used to avoid repetition in the constant, naming this `emptyAnnotations` would be a bit clearer. Personally I’d just use a literal `map[string]string{}` in data.

Either is fine.

---

… in all three formats.